.github/CONTRIBUTING.md

```diff
@@ -41,6 +45,18 @@ Before you submit your pull request consider the following guidelines:
 
 * Search [GitHub](https://github.com/runelite/runelite/pulls) for an open or closed Pull Request
   that relates to your submission. You don't want to duplicate effort.
+* Fork this repo.
```

Comment:
I think most users who would have to read this document would not understand what this means. Maybe link to https://help.github.com/articles/fork-a-repo/. And https://help.github.com/articles/cloning-a-repository/.